packages/client-config/src/client-config-schema/schema_v1.json

```diff
@@ -105,10 +105,14 @@
               "minItems": 0,
               "uniqueItems": true
             },
-            "domain": {
+            "cognito_domain": {
               "description": "Cognito Domain used for identity providers",
```

Comment:
nit: just glancing at this, "Custom" and "Cognito" look very similar. Maybe change this description to `Autogenerated domain used for identity providers`?

Reply:
Updated.